util/network/src/client_version.rs

```diff
@@ -22,8 +22,8 @@ use semver::Version;
 use std::fmt;
 
 /// Parity client string prefix
-const LEGACY_CLIENT_ID_PREFIX: &str = "Parity";
-const PARITY_CLIENT_ID_PREFIX: &str = "Parity-Ethereum";
+const LEGACY_CLIENT_ID_PREFIX: &str = "Parity-Ethereum";
+const PARITY_CLIENT_ID_PREFIX: &str = "Open-Ethereum";
```

Comment:
here we change the advertised version in the network

Reply:
Should we change the name of the `const` perhaps?